community/gegl/APKBUILD

```diff
@@ -1,46 +1,39 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gegl
-pkgver=0.2.0
-pkgrel=5
+pkgver=0.4.2
+pkgrel=0
 pkgdesc="Graph based image processing framework"
 url="http://www.gegl.org/"
 arch="all"
 license="GPL-3.0 LGPL-3.0"
 makedepends="babl-dev libpng-dev libjpeg-turbo-dev gtk+-dev librsvg-dev
-	lua5.1-dev jasper-dev exiv2-dev"
+	lua5.1-dev jasper-dev exiv2-dev json-glib-dev"
 subpackages="$pkgname-dev $pkgname-lang"
-source="http://ftp.gimp.org/pub/$pkgname/${pkgver%.*}/$pkgname-$pkgver.tar.bz2
-	"
-
-depends_dev="babl-dev glib-dev"
-_builddir="$srcdir"/$pkgname-$pkgver
+source="http://ftp.gimp.org/pub/$pkgname/${pkgver%.*}/$pkgname-$pkgver.tar.bz2"
+builddir="$srcdir"/$pkgname-$pkgver
 prepare() {
```

Comment:
We nowadays have a [default prepare]( https://git.alpinelinux.org/cgit/abuild/tree/abuild.in#n620)rule for applying patches. This function can be deleted entirely.